api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java

```diff
@@ -72,41 +72,49 @@
       "select distinct person_id, entry_date, concept_id\n"
           + "from `${projectId}.${dataSetId}.cb_search_all_events`\n"
           + "where ";
+  private static final String STANDARD_SQL = "is_standard = %s";
+  private static final String CONCEPT_ID_UNNEST_SQL = "concept_id in unnest(%s)";
+  private static final String CONCEPT_ID_IN_SQL = "concept_id in";
   private static final String STANDARD_OR_SOURCE_SQL =
-      "is_standard = %s and concept_id in unnest(%s)\n";
-  private static final String PARENT_STANDARD_OR_SOURCE_SQL =
-      "is_standard = %s and concept_id in (select distinct c.concept_id\n"
+      STANDARD_SQL + AND + CONCEPT_ID_UNNEST_SQL + "\n";
+  public static final String CHILD_LOOKUP_SQL =
+      " (select distinct c.concept_id\n"
           + "from `${projectId}.${dataSetId}.cb_criteria` c\n"
-          + "join (${childLookup}) a\n"
+          + "join (select cast(cr.id as string) as id\n"
+          + "from `${projectId}.${dataSetId}.cb_criteria` cr\n"
+          + "where domain_id = %s\n"
+          + "and is_standard = %s\n"
+          + "and concept_id in unnest(%s)\n"
```

Comment:
Do you think a part of this can be replaced by STANDARD_OR_SOURCE_SQL or will it be too confusing to read?

Reply:
Not sure we should tie them together, since there used in different parts of the sql generation. I've only tried to reuse sql parts for things generated in the same context.